br/pkg/lightning/backend/external/bench_test.go

```diff
@@ -314,7 +314,8 @@ func createEvenlyDistributedFiles(
 	kvCnt := 0
 	for i := 0; i < fileCount; i++ {
 		builder := NewWriterBuilder().
-			SetMemorySizeLimit(uint64(float64(fileSize) * 1.1))
+			SetBlockSize(10 * 1024 * 1024).
+			SetMemorySizeLimit(uint64(float64(fileSize) * 1.2))
```

Comment:
why here change from 1.1 to 1.2? magic number?

Reply:
https://github.com/pingcap/tidb/pull/48275/commits/5375d41fb9f83fe59d97cae39393ca6809857bf8 reverted. I forget why 😂